scripts/train-mask-align/preprocess_europarl.sh

```diff
@@ -2,16 +2,6 @@
 
 set -ex
 
-mkdir -p corpus-es
-cd corpus-es/
```

Comment:
The `cd corpus-es/` should be kept, right?

Reply:
When you run a script, it doesn't leave your current fit changed if it changes internally. Unless you `source` it (or with `.`).